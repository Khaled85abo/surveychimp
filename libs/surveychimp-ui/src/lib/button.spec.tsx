import { render } from '@testing-library/react';
import Button from './button';

describe("Button", () => {
    it("should match snapshot", () => {
        const renderResult = render(<Button label="Klicka hÃ¤r!" />);
        expect(renderResult.baseElement).toMatchSnapshot();
    })
})