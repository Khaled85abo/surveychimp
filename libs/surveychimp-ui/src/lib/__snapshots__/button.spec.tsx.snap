// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot 1`] = `
<body>
  <div>
    <button
      class="button__StyledButton-sc-156p7kj-0 iPHCQY"
    >
      Klicka h√§r!
    </button>
  </div>
</body>
`;
